bin/require-from-cwd.js

```diff
@@ -0,0 +1,6 @@
+const resolve = require( "resolve" );
```

Comment:
Not related to this PR, but we seem to have grown a fair number of files in ‘bin/‘ that are not intended as standalone CLI entry points. We should consider moving these to ‘src/‘ instead.

Reply:
:+1: agreed, though I would like to ensure we keep some clear separation between CLI code and library code (just means we likely need `src/cli` or something)

Reply:
Sounds good to me!